components/brave_wallet_ui/utils/account-utils.ts

```diff
@@ -99,7 +99,23 @@ export const findAccountByAccountId = (
   if (!accounts) {
     return undefined
   }
-  return accounts.entities[entityIdFromAccountId(accountId)]
+
```

Comment:
@josheleonard Need your eyes on this block.

Reply:
LGTM